utils/vlog/tests/dsp_combinational/golden.pb_type.xml

```diff
@@ -3,7 +3,7 @@
   <input name="b" num_pins="32"/>
   <input name="m" num_pins="1"/>
   <output name="out" num_pins="64"/>
-  <delay_constant in_port="dsp_combinational.a" max="30e-12" out_port="dsp_combinational.out"/>
-  <delay_constant in_port="dsp_combinational.b" max="30e-12" out_port="dsp_combinational.out"/>
```

Comment:
It is legal to leave off the module name?

Reply:
Explained previously.